src/pre_defined_classes/mathematical_program/hard/q5/LeftRight.java

```diff
@@ -0,0 +1,26 @@
+package pre_defined_classes.mathematical_program.hard.q5;
+
+/*
+Make a Java program to check the following trigonometric identity by displaying the value of the left hand side of the
+equation as well as the right hand side of the equation:
+sin( A + B ) = sin( A )cos( B ) + cos( A )sin( B ) where A and B are angles in radians.
+
+Distractors:
+int A = 2.56;
+int B = 1.4312;
+float leftHandSide = Math.sinh( A+B );
+double leftHandSide = sin( A + B );
+double rightHandSide = sin( A ) * cos( B ) + cos( A ) * sin( B );
+double rightHandSide = Math.sin( A )Math.cos( B ) + Math.cos( B )Math.sin( B );
+ */
```

Comment:
I think you should add a sample output so they know to print left hand side first

Reply:
My clause doesn't check for a specific double output, just a double, and using the delta at all basically means it registers both answers as right for both asserts, so it doesn't really matter

Reply:
No I meant the string for this one, you have the lefthand side one first, and if they switch the order around it will not work